src/svg/qsvggenerator.cpp

```diff
@@ -934,6 +942,17 @@ bool QSvgPaintEngine::end()
     Q_D(QSvgPaintEngine);
 
     d->stream->setString(&d->defs);
+
+	for (auto it = clip_path_to_id.begin(); it != clip_path_to_id.end(); it++)
+	{
+		std::string path = it->first;
+		int path_id = it->second;
+
+		*d->stream << "<clipPath id=\"clip" << path_id << "\">" << '\n';
+		*d->stream << "\t" << path.c_str() << " \"/> \n";
```

Comment:
Space before newline.  Also, the other lines stream the newline separately, whereas this one does not.  Choose one way or the other for consistency.